packs/puppet/actions/run_agent.py

```diff
@@ -10,15 +10,15 @@
 
 
 class PuppetRunAgentAction(PuppetBaseAction):
-    def run(self, server=None, certname=None, daemonize=False, onetime=True,
-            debug=None):
+    def run(self, server=None, certname=None, environment=None,
```

Comment:
Hm, am I missing something? I don't actually see `environment` argument being used in the method body.